e2e/kubelet/generate-kubelet-flags.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euox pipefail
+
+retrycmd_get_tarball() {
+    tar_retries=$1; wait_sleep=$2; tarball=$3; url=$4
+    echo "${tar_retries} retries"
+    for i in $(seq 1 $tar_retries); do
+        tar -tzf $tarball && break || \
+        if [ $i -eq $tar_retries ]; then
+            return 1
+        else
+            timeout 60 curl -fsSL $url -o $tarball
+            sleep $wait_sleep
+        fi
+    done
+}
+
+sudo swapoff -a
+sudo ctr version
+sudo systemctl status containerd
+
+sed '$ d' parts/linux/cloud-init/artifacts/manifest.json > temporary_manifest.json
```

Comment:
should we consider doing all of it in Go?

Reply:
can do as part of installing containerd ourselves